flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/Column.java

```diff
@@ -108,6 +119,13 @@ public String asSummaryString() {
                             sb.append(" ");
                             sb.append(e);
                         });
+        getComment()
+                .ifPresent(
+                        c -> {
+                            sb.append(" COMMENT '");
+                            sb.append(c);
```

Comment:
We should wrap this with `EncodingUtils#escapeSingleQuotes`.

Reply:
@Airblader  thanks for the review, I update the code, and add the related test case.